extensions/blocks/instagram-gallery/edit.js

```diff
@@ -60,18 +60,31 @@ const InstagramGalleryEdit = props => {
 				access_token: accessToken,
 				count,
 			} ),
-		} ).then( response => {
+		} ).then( ( { external_name: externalName, images: imageList } ) => {
```

Comment:
I wonder if there's an optimisation to be done here. If the count has reduced, we shouldn't need to call the API. Perhaps that's a step too far for this PR.

Also, could we use async/await here to reduce the nesting?